tests/integration/workflow.py

```diff
@@ -244,3 +257,41 @@ def run(self, *args, **kwargs):
             )
         )
         futures.wait(future)
+
+
+class WorkflowWithTooBigData(Workflow):
+    name = "example"
+    version = "example"
+    task_list = "example"
+
+    def run(self, *args, **kwargs):
+        group = Chain(break_on_failure=False)
+        group.append(pingpong_activity, "ab", JUMBO_FIELDS_MAX_SIZE)
+        group.append(pingpong_activity, "ab" * JUMBO_FIELDS_MAX_SIZE, 0.25)
```

Comment:
I'm guessing the first line tests a long output, and the second one a long input. This could be made clearer with comments.

The fact that the base string is 2 chars is also obscure. This ensures that the first activity will be well above the limit, but forces the 2e activity to use 0.25 to divide by a greater number and ensure that it's below the limit.

I think we could make this whole thing clearer. Better naming, clearer tests.

Reply:
We're testing 2 things at once. When we trigger the expected failure, how do we know it's the correct one that failed ?